### PR TITLE
Add python3-mediapipe-pip

### PR DESCRIPTION
**Pull request by viktorlorentz**
Please add the following dependency to the rosdep database.

## Package name:

python-mediapipe-pip

## Package Upstream Source:

https://github.com/google/mediapipe

## Purpose of using this:

This adds mediapipe solutions to rosdep. Mediapipe solutions offer an easy and performant way to build pose tracking and other ml based applications.

Distro packaging links:

## Links to Distribution Packages

PyPi: https://pypi.org/project/mediapipe/

### CODE REVIEW

**Review by mabelzhang — APPROVED** [review]

**Comment by mabelzhang** [comment]
Looks like there's an error in the tests from `test/rosdep_repo_check/test_rosdep_repo_check.py`. Could you fix the error? I'm not able to run the test locally for some weird yaml reason. Maybe try merging from `master`.

**Comment by mabelzhang** [comment]
Okay merged from `master`. That should fix the tests.

**Comment by viktorlorentz** [comment]
@mabelzhang thx for fixing the error. LGTM now!

File: rosdep/python.yaml [hunk 1]
```diff
@@ -2632,6 +2632,19 @@ python-mechanize:
   gentoo: [dev-python/mechanize]
   nixos: [pythonPackages.mechanize]
   ubuntu: [python-mechanize]
+python-mediapipe-pip:
```
Comment (ivanpauno) [inline rosdep/python.yaml:2635]: did you wanted to add the python2 version or actually one for python3?
according to the [contributing guidelines](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#python-rules), the prefix should be `python3` and not `python` in that case.

AFAIS `mediapipe` never supported python2

**Review by ivanpauno — CHANGES_REQUESTED** [review]

File: rosdep/python.yaml (hunk 1 above)
Comment (viktorlorentz) [reply rosdep/python.yaml]: Thy for the hint. I updated the prefix accordingly.

File: rosdep/python.yaml [hunk 2]
```diff
@@ -2632,6 +2632,19 @@ python-mechanize:
   gentoo: [dev-python/mechanize]
   nixos: [pythonPackages.mechanize]
   ubuntu: [python-mechanize]
+python3-mediapipe-pip:
```
Comment (ivanpauno) [inline rosdep/python.yaml:2635]: you need to use alpha order to make the tests pass.
You can use rosdistro_reformat, which is available after installing python3-rosdistro (example [here](http://wiki.ros.org/rosdistro/Tutorials/Indexing%20Your%20ROS%20Repository%20for%20Documentation%20Generation#Steps_to_have_your_repository_added_to_our_indexer))

Comment (viktorlorentz) [reply rosdep/python.yaml]: Rookie mistake... Its sorted now.

Comment (ivanpauno) [reply rosdep/python.yaml]: no problem, thanks for updating!

**Review by ivanpauno — APPROVED** [review]

**Review by hidmic — APPROVED** [review]

**Comment by hidmic** [comment]
Thanks for your contribution and patience @viktorlorentz.

**Pull request merged**